modules/architecture/pages/traditional-rhnsd.adoc

```diff
@@ -1,7 +1,10 @@
 [[bp.contact.methods.rhnsd]]
 = Traditional Contact Method (rhnsd)
 
+For background information, see xref:architecture:rhnsd.adoc[].
+*rhnsd* is used non-systemd systems; on later systems, a systemd timer ([systemitem]``rhnsd.timer``) is used.
 
+The daemon is NOT started by "rhnsd.service", but by /etc/init.d/rhnsd
```

Comment:
```suggestion
The daemon is started by /etc/init.d/rhnsd, it does not use rhnsd.service. 
```

Two general comments here: put the most important/correct information first (people skim tech docs, and are more likely to issue a command that appears first in a sentence). Adding a few extra words ("it does not" rather than "not") makes the second clause more obviously a negative statement.

Reply:
trying to remember it ;)